app/models/edition.rb

```diff
@@ -445,6 +445,10 @@ def latest_link_check_report
     link_check_reports.last
   end
 
+  def hide_chapter_navigation
```

Comment:
I don't think you need this method here. I think adding the field to `GuideEdition` is enough.